python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -1658,6 +1658,90 @@ def export_torchscript_file(self, dirname="fp32_torchscript",
                 self.build_jit()
             InferenceOptimizer.save(self.accelerated_model, dirname)
 
+    def export_torchscript_module(self, tsdata, path_dir=None, drop_dtcol=True):
```

Comment:
should we just make this in `export_torchscript_file` and add some optional parameters for it?

Reply:
Sounds good, I will have a try.

Reply:
> should we just make this in `export_torchscript_file` and add some optional parameters for it?

have put the code in `export_torchscript_file`, please take a look.